Classes/Private/Manager/Objects/OCTActiveFile+Variants.h

```diff
@@ -0,0 +1,58 @@
+//
+//  OCTActiveFile+Variants.h
+//  objcTox
+//
+//  Created by stal on 9/7/2015.
+//  Copyright © 2015 Zodiac Labs. All rights reserved.
+//
+
+#import "OCTSubmanagerFiles.h"
+#import "OCTActiveFile.h"
+
+#define AVERAGE_SECONDS 10
+
+@class OCTMessageFile;
+
+@interface OCTActiveFile ()
+
+@property (weak)      OCTSubmanagerFiles *fileManager;
+// Observation: I'd rather not hold on to a Realm object. Who knows what could
+//              happen.
+@property (strong)    OCTMessageFile     *fileMessage;
+@property             OCTToxFriendNumber friendNumber;
+@property             OCTToxFileSize fileSize;
+@property (readwrite) OCTToxFileSize bytesMoved;
+@property             BOOL isConduitOpen;
+
+@property             time_t lastCountedTime;
+@property             OCTToxFileSize     *transferRateCounters;
+@property             long rollingIndex;
+
+@property (strong)    OCTFileNotificationBlock notificationBlock;
+
+/* Helpful if a bit unclean. */
+- (id<OCTFileConduit>)conduit;
+
+- (void)sendProgressUpdateNow;
+- (void)control:(OCTToxFileControl)ctl;
+- (void)interrupt;
+
+@end
+
+@interface OCTActiveOutboundFile : OCTActiveFile
```

Comment:
Messages use "outgoing" and "incoming" words. Maybe be consistent and use `OCTActiveOutgoingFile`, `OCTActiveIncomingFile`?